topics/climate/tutorials/pangeo-notebook/tutorial.md

```diff
@@ -0,0 +1,714 @@
+---
+layout: tutorial_hands_on
+
+title: Pangeo Notebook in Galaxy - Introduction to Xarray
+zenodo_link: 'https://doi.org/10.5281/zenodo.5805953'
+requirements:
+- 
+  type: internal
+  topic_name: data-science
+  tutorials:
+    - python-basics
+- 
+  type: internal
+  topic_name: data-science
+  tutorials:
+    - python-plotting
+    - python-advanced-np-pd
+- 
+  type: "external"
+  title: Data Carpentry Geospatial Workshop
+  link: https://datacarpentry.org/geospatial-workshop/
+follow_up_training:
+- 
+  type: "external"
+  title: Pangeo Tutorial Gallery
+  link: http://gallery.pangeo.io/repos/pangeo-data/pangeo-tutorial-gallery/index.html
+- 
+  type: "external"
+  title: Pangeo Gallery
+  link: http://gallery.pangeo.io/index.html
+questions:
+- What is Pangeo notebook?
+- How to start Pangeo Notebook in Galaxy?
+- What are the main software components of the Pangeo ecosystem?
+- What is Xarray?
+- How to manipulate Xarrays?
+- How to print metadata information?
+- How to make a selection?
+- How to visualize?
+- How to filter? 
+- How to make reduction operations (mean, max, min)?
+- How to resample my data?
+- Where to go next?
+objectives:
+- Learn to get metadata information using Xarray Galaxy Tools
+- Learn to select data
+- Learn to visualize data
+- Learn to filter, make reduction operations (mean, max, min)
+- Learn to resample my data
+- Learn to cite and contribute to Pangeo
+time_estimation: 1H
+key_points:
+- Pangeo ecosystem enables big data analysis in geosciences
+- Xarray is an important Python package for big data analysis in geosciences 
+- Xarray can be used to read, select, mask and plot netCDF data
+- Xarray can also be used to perform global operations such as mean, max, min or resampling data
+tags:
+  - pangeo
+  - interactive-tools
+  - jupyter-notebook
+contributors:
+- annefou
+notebook:
+  language: python
+  snippet: topics/climate/tutorials/pangeo-notebook/preamble.md
+---
+
+
+# Introduction
+{:.no_toc}
+
+In this tutorial, we will learn about [Xarray](https://xarray.pydata.org/), one of the most used Python library from the [Pangeo](https://pangeo.io/) ecosystem. 
+
+We will be using data from [Copernicus Atmosphere Monitoring Service](https://ads.atmosphere.copernicus.eu/)
+and more precisely PM2.5 ([Particle Matter < 2.5 μm](https://en.wikipedia.org/wiki/Particulates#Size,_shape_and_solubility_matter)) 4 days forecast from December, 22 2021. Parallel data analysis with Pangeo is not covered in this tutorial. 
+
+> ### {% icon comment %} Remark
+>
+> This tutorial uses data on a regular latitude-longitude grid. More complex and irregular grids are not discussed in this tutorial. In addition,
+> this tutorial is not meant to cover all the different possibilities offered by Xarrays but shows functionalities we find useful for day to day
+> analysis.
+>
+{: .comment}
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+
+# Analysis
```

Comment:
maybe a short paragraph mentioning that we will use the`interactive_tool_pangeo_notebook`  tool Then launching a Python 3 kernel?

Reply:
Yes it is added (you need to look at the rendered version). Some kind of "magic":
![image](https://user-images.githubusercontent.com/8168508/154761386-d78351e3-d6f1-4b06-9f63-0df2583fec15.png)

Reply:
perfect! Duno why I don't see mention of these magic content on the md file.. but ok, thanks for the screenshots! I can continue!!

Reply:
.. I was searching on zenodo in case of.. but don't see it on the Galaxy training dedicated space, now I understand why ;) data is not on this Zenodo space! OK for me!!!

Reply:
I am not fully sure (that's why I say it is magic!). I think it is based on the the tags we define at the top.